draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -2273,6 +2273,15 @@ Note that the verifier will know the elements in the left side of the above equa
 * Added MapMessageToScalar test vectors
 * Fix typo in ciphersuite name
 
+-02
+
+* Variety of editiorial clarifications
+* Clarified integer endianness
+* Revised the encode for hash operation
+* Shifted to using CSPRING instead of PRF
+* Removed total number of messages from proof verify operation
+* Added deterministic proof fixtures
+
```

Comment:
```suggestion
* Shifted to multiple CSPRNG calls to calculate random elements, instead of expand_message
* Updated hash_to_scalar to a single output

```
Some small additions.